docs/tfengine/schemas/audit.md

```diff
@@ -7,14 +7,14 @@
 | Property | Description | Type | Required | Default | Pattern |
 | -------- | ----------- | ---- | -------- | ------- | ------- |
 | additional_filters | Additional filters for log collection and export. List entries will be        concatenated by "OR" operator. Refer to        <https://cloud.google.com/logging/docs/view/query-library> for query syntax.        Need to escape \ and " to preserve them in the final filter strings.        See example usages under "examples/tfengine/".        Logs with filter `"logName:\"logs/cloudaudit.googleapis.com\""` is always enabled. | array(string) | false | - | - |
-| auditors_group | This group will be granted viewer access to the audit log dataset and        bucket as well as security reviewer permission on the root resource        specified. | string | false | - | - |
+| auditors_group | This group will be granted viewer access to the audit log dataset and        bucket as well as security reviewer permission on the root resource        specified. | string | true | - | - |
 | bigquery_location | Location of logs bigquery dataset. | string | false | - | - |
-| logs_bigquery_dataset | Bigquery Dataset to host audit logs for 1 year. Useful for querying recent activity. | object | false | - | - |
-| logs_bigquery_dataset.dataset_id | ID of Bigquery Dataset. | string | false | - | - |
-| logs_bigquery_dataset.sink_name | Name of the logs sink, default to "bigquery-audit-logs-sink". | string | false | - | - |
-| logs_storage_bucket | GCS bucket to host audit logs for 7 years. Useful for HIPAA audit log retention requirements. | object | false | - | - |
-| logs_storage_bucket.name | Name of GCS bucket. | string | false | - | - |
-| logs_storage_bucket.sink_name | Name of the logs sink, default to "storage-audit-logs-sink". | string | false | - | - |
+| logs_bigquery_dataset | Bigquery Dataset to host audit logs for 1 year. Useful for querying recent activity. | object | true | - | - |
+| logs_bigquery_dataset.dataset_id | ID of Bigquery Dataset. | string | true | - | - |
+| logs_bigquery_dataset.sink_name | Name of the logs sink. | string | false | bigquery-audit-logs-sink | - |
+| logs_storage_bucket | GCS bucket to host audit logs for 7 years. Useful for HIPAA audit log retention requirements. | object | true | - | - |
+| logs_storage_bucket.name | Name of GCS bucket. | string | true | - | - |
+| logs_storage_bucket.sink_name | Name of the logs sink. | string | false | storage-audit-logs-sink | - |
 | parent_id | ID of parent GCP resource to apply the policy.        Can be one of the organization ID or folder ID according to parent_type. | string | false | - | ^[0-9]{8,25}$ |
 | parent_type | Type of parent GCP resource to apply the policy.        Must be one of 'organization' or 'folder'." | string | false | - | ^organization\|folder$ |
 | project | Config of project to host auditing resources | object | false | - | - |
```

Comment:
I didn't see these `project` variables used in the audit recipe. I suppose they're inherited from `project` recipe somehow, since there is a `module.project.project_id` at the `main.tf`

Two questions:

1. If it's always inherited, I suppose it's here to override the default since it's not required, isn't?
2. I saw some other inherited variables that are not pretty clear when they're required or not. Most of them seem required since it's templating syntax is like they were, but I'm not sure if I have to add them as required.